content/_data/sponsors.js

```diff
@@ -0,0 +1,132 @@
+// https://opencollective.com/oddbird-open-source/members/all.json
+import eleventyFetch from '@11ty/eleventy-fetch';
+import { config } from 'dotenv';
+import { groupBy } from 'lodash-es';
+
+// eslint-disable-next-line no-process-env
+if (!process.env.GITHUB_KEY) {
+  // Load .env variables with dotenv
+  config();
+}
+// eslint-disable-next-line no-process-env
+const GITHUB_KEY = process.env.GITHUB_KEY;
+
+const FilteredProfiles = [
+  // if there are backers we need to exclude…
+];
+
+const TIERS = ['Great Horned Owl', 'Blue-Footed Booby', 'Common Loon'];
+
+const getDefaultAvatarUrl = (url) => {
+  const slug = url.split('/').at(-1);
+  return slug ? `https://images.opencollective.com/${slug}/avatar.png` : null;
+};
+
+const loadGithubSponsors = async () => {
+  if (!GITHUB_KEY) {
+    // eslint-disable-next-line no-console
+    console.error('Github sponsors not loaded- set GITHUB_KEY in .env.');
```

Comment:
I don't feel like this is a requirement for local dev, so opted to put the usage directions here on failure as opposed to in a README.